src/quilla/reports/visual_parity_report.py

```diff
@@ -0,0 +1,77 @@
+
+from quilla.common.enums import (
+    XPathValidationStates,
+    ValidationTypes
+)
+from quilla.reports.validation_report import ValidationReport
+
+
+class VisualParityReport(ValidationReport):
+    '''
+    Report type for VisualParity validation state
+
+    Args:
+        target: The validation target
```

Comment:
Are you planning to add running ID here or on a higher level for all types of Quilla test?

Reply:
I'll be addressing the running ID in a different PR since it has to be integrated into all of Quilla.

Right now each quilla test would get a new running ID per run, so multiple quilla tests as part of one testing suite would get multiple running IDs per run, and it might make more sense to consider how to have it so they all share one running ID. I'll add an issue for this

Reply:
Added #38 for this